USAGE

```diff
@@ -0,0 +1,45 @@
+FreeWater and FW corrected DTI metrics pipeline
+===============================================
+
+Run the FreeWater pipeline, including FW-corrected DTI metrics estimation
+
+USAGE
+
+nextflow run main.nf [OPTIONAL_ARGUMENTS] --input
+
+DESCRIPTION
+
+    --input=/path/to/[root]             Root folder containing multiple subjects, with the following files.
+
+                                        [input]
+                                        ├── sub1
+                                        │   ├── brain_mask.nii.gz
+                                        │   ├── bval
+                                        │   ├── bvec
+                                        │   └── dwi.nii.gz
+                                        └── sub2
+                                            ├── brain_mask.nii.gz
+                                            ├── bval
+                                            ├── bvec
+                                            └── dwi.nii.gz
+
+Files can come out of the TractoFlow pipeline, including all preprocessing steps. If so, you should select the
+DWI in Resample_DWI, brain mask in Resample_B0, bval/bvec from Eddy_Topup.
```

Comment:
Can you reformat exactly as NODDI flow please for uniformity.

If files come out of the Tractoflow pipeline, you should take the following source:
- dwi.nii.gz: from Resample_DWI
- brain_mask.nii.gz: from Resample_B0
- bval/bvec: from Eddy_Topup (or Eddy)